doc/release-notes/10162-globus-support.md

```diff
@@ -0,0 +1,19 @@
+Globus support in Dataverse has been expanded to include support for using file-based Globus endpoints, including the case where files are stored on tape and are not immediately accessible, 
+and for referencing files stored on remote Globus endpoints. Support for using the Globus S3 Connector with an S3 store has been retained but requires changes to the Dataverse configuration. 
```

Comment:
"requires changes to the Dataverse configuration" - does this warrant spelling out these required changes in the upgrade instruction? Or is this a non-issue, since no other instances are relying on the functionality yet?

Reply:
Not really - there aren't many people, but the details are basically delete what you had before and do it the new way (it was basically global rather than per-store now).